data_profiler/tests/data/csv/names-col-empty.txt

```diff
@@ -0,0 +1,19 @@
+NAMES
+George Washington
+
+
+
+
+John Adams
+Thomas Jefferson
+James Madison
+
+
+
+
+
+
+
+
+James Monroe
+John Adams
```

Comment:
do we have files with blank lines at the last lines?

Reply:
That's actually how I discovered this issue (big file of newline characters, nothing else).

I can add it to the end of this file.